src/System.Data.SqlClient/src/Interop/SNINativeMethodWrapper.Unix.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Data.SqlClient.SNI;
+
+namespace System.Data.SqlClient
+{
+    internal static class SNINativeMethodWrapper
+    {
+        internal enum SniSpecialErrors : uint
```

Comment:
There are lots of references to SNINativeMethodWrapper in TdsParser and other places.